test/unit/lib/plugins/aws/package/compile/events/apiGateway/lib/validate.test.js

```diff
@@ -412,37 +453,28 @@ describe('#validate()', () => {
     expect(authorizer.identityValidationExpression).to.equal('foo');
   });
 
-  it('should process cors defaults', () => {
-    awsCompileApigEvents.serverless.service.functions = {
-      first: {
-        events: [
-          {
-            http: {
-              method: 'POST',
-              path: '/foo/bar',
-              cors: true,
-            },
-          },
-        ],
-      },
-    };
-
-    const validated = awsCompileApigEvents.validate();
-    expect(validated.events).to.be.an('Array').with.length(1);
-    expect(validated.events[0].http.cors).to.deep.equal({
-      headers: [
+  it('should process cors defaults', async () => {
```

Comment:
It looks great - one last thing will be to move the `runServerless` based test to a separate `describe` section which is named after the path to this test file, similarly to e.g. https://github.com/serverless/serverless/blob/1dfad8f13ca73ee088b88a2c2c955b23af640f64/test/unit/lib/plugins/aws/deployFunction.test.js#L313

Reply:
Thanks for the clear description! Updated